### PR TITLE
fix module names to reflect Java EE origin

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by rotty3000** [comment]
@martin-g @markt-asf does this look reasonable? I do need to apply the .api change to master separately.

File: res/bnd/tomcat-embed-core.jar.tmp.bnd [hunk 1]
```diff
@@ -131,15 +131,15 @@ Provide-Capability: \
...
-    jakarta.ejb-api;static=true,\
-    jakarta.mail;static=true,\
-    jakarta.persistence-api;static=true,\
-    jakarta.xml.rpc-api;static=true,\
-    java.xml.ws;static=true
+    javax.ejb.api;static=true,\
+    javax.mail;static=true,\
+    javax.persistence.api;static=true,\
+    javax.xml.rpc.api;static=true,\
+    javax.xml.ws.api;static=true
```
Comment (markt-asf) [inline res/bnd/tomcat-embed-core.jar.tmp.bnd:145]: This looks to have a module name of `javax.xml.ws` in Java EE 8

File: res/bnd/tomcat-embed-core.jar.tmp.bnd [hunk 2]
```diff
@@ -131,15 +131,15 @@ Provide-Capability: \
...
 -jpms-module-info-options: \
     ${module.name};substitute=tomcat-embed-core,\
-    jakarta.ejb-api;static=true,\
-    jakarta.mail;static=true,\
-    jakarta.persistence-api;static=true,\
-    jakarta.xml.rpc-api;static=true,\
-    java.xml.ws;static=true
+    javax.ejb.api;static=true,\
+    javax.mail;static=true,\
+    javax.persistence.api;static=true,\
```
Comment (markt-asf) [inline res/bnd/tomcat-embed-core.jar.tmp.bnd:143]: This looks to have a module name of `java.persistence` in Java EE 8

File: res/bnd/annotations-api.jar.tmp.bnd [hunk 3]
```diff
@@ -35,9 +35,9 @@ Provide-Capability: \
     Specification-Version=1.3
 
 -jpms-module-info: \
-    jakarta.annotation;\
+    javax.annotation.api;\
         access=32;\
         version=${annotation.spec.version}
 -jpms-module-info-options: \
-    jakarta.annotation;\
+    javax.annotation.api;\
```
Comment (markt-asf) [inline res/bnd/annotations-api.jar.tmp.bnd:42]: The Java EE 8 JAR uses `java.annotation` as the module name

File: res/bnd/catalina.jar.tmp.bnd [hunk 4]
```diff
@@ -55,15 +55,15 @@ Export-Package: \
         access=32;\
         version=${Bundle-Version};\
         modules='\
-            jakarta.ejb-api,\
-            jakarta.mail,\
-            jakarta.persistence-api,\
-            jakarta.xml.rpc-api,\
+            javax.ejb.api,\
+            javax.mail,\
```
[the remark this answers is not in the document]
Comment (rotty3000) [reply res/bnd/catalina.jar.tmp.bnd]: fixed

File: res/bnd/tomcat-embed-core.jar.tmp.bnd (hunk 1 above)
Comment (rotty3000) [reply res/bnd/tomcat-embed-core.jar.tmp.bnd]: fixed

File: res/bnd/annotations-api.jar.tmp.bnd (hunk 3 above)
Comment (rotty3000) [reply res/bnd/annotations-api.jar.tmp.bnd]: fixed

File: res/bnd/tomcat-embed-core.jar.tmp.bnd (hunk 2 above)
Comment (rotty3000) [reply res/bnd/tomcat-embed-core.jar.tmp.bnd]: fixed

**Pull request merged**